manim/mobject/geometry.py

```diff
@@ -613,6 +616,31 @@ def __init__(self, *args, **kwargs):
         self.set_stroke_width_from_length()
 
     def scale(self, factor, **kwargs):
```

Comment:
Suggestion: I think this behavior should be the default, but it should be optional. Can we add a boolean that would allow the user to propagate the scale() call to the tips?

Reply:
Good call! cfb2cdaab5909b58859bc769cb914693b6bb951a